ivre/passive.py

```diff
@@ -227,9 +231,13 @@ def _prepare_rec(spec, ignorenets, neverignore):
                                          spec, exc_info=True)
             elif ntlm._is_ntlm_message(value):
                 # NTLM_CHALLENGE
-                auth = utils.decode_b64(value.split(' ', 1)[1].encode())
-                spec['value'] = "NTLM %s" % \
-                    ntlm._ntlm_dict2string(ntlm.ntlm_extract_info(auth))
+                try:
+                    auth = utils.decode_b64(value.split(' ', 1)[1].encode())
```

Comment:
You should use `None` instead of `' '` as separator since it will work if two space chars (or a tab) are used:
```suggestion
                     auth = utils.decode_b64(value.split(None, 1)[1].encode()) 
```